docs/control-structures.rst

```diff
@@ -51,40 +56,45 @@ write::
 
 The names of output parameters can be omitted.
 The output values can also be specified using ``return`` statements.
-The ``return`` statements are also capable of returning multiple
-values, see :ref:`multi-return`.
+
+The ``return`` statements are also capable of returning multiple values using ``return (v0, v1, ..., vn)``. The number of parameters must equal the number of output parameters.
```

Comment:
Is it obvious that the types also have to match?

Reply:
@chriseth No… Fixed.